.travis.yml

```diff
@@ -11,7 +11,9 @@ matrix:
 notifications:
   email: false
 before_script:
+  - cargo +nightly install rustfmt-nightly --force
```

Comment:
Is there a nice way to not run this on stable rust in Travis so it doesn't falsely fail the build?

Reply:
The release version of `rustfmt` is unmaintained and depends on `syntex`, which is now deprecated in favour of the `libsyntax` part of the rust nightly compiler. The plan is to standardise that in the long term, but for now we live in an awkward world where `rustfmt` and `rustfmt-nightly` have diverged so much that the formatted output differs greatly. Since the old formatting rules no longer apply (and many have incorrect output, fixed in the `rustfmt-nightly` version), using nightly is a necessary evil for now if we want to have consistency with future releases.